devices/HumanWrenchProvider/HumanWrenchProvider.cpp

```diff
@@ -845,19 +1264,22 @@ bool HumanWrenchProvider::attach(yarp::dev::PolyDriver* poly)
     // Get the device name from the driver
     const std::string deviceName = poly->getValue("device").asString();
 
-    if (deviceName == "human_state_provider") {
+    yInfo()<<LogPrefix<<"Attaching "<< deviceName;
+
+    if (deviceName == "human_state_provider" || deviceName == "human_state_remapper" ) {
 
         // Attach IHumanState interface from HumanStateProvider
         if (pImpl->iHumanState || !poly->view(pImpl->iHumanState) || !pImpl->iHumanState) {
             yError() << LogPrefix << "Failed to view IHumanState interface from the polydriver";
             return false;
         }
 
-        // Check the interface
-        if (pImpl->iHumanState->getNumberOfJoints() == 0
+        //TODO Check the interface
+        int count = 1000000;
+        while (pImpl->iHumanState->getNumberOfJoints() == 0
                 || pImpl->iHumanState->getNumberOfJoints() != pImpl->iHumanState->getJointNames().size()) {
-            yError() << "The IHumanState interface might not be ready";
-            return false;
+            yError() << LogPrefix<<"The IHumanState interface might not be ready";
+            if(count--==0) return false;
```

Comment:
For readability, can we simplify as:
```suggestion
            count--;
            if(count==0) return false;
```